vendor/github.com/coreos/go-oidc/oidc/provider.go

```diff
@@ -353,9 +353,6 @@ func (p ProviderConfig) Valid() error {
 	if !contains(p.IDTokenSigningAlgValues, "RS256") {
 		return errors.New("id_token_signing_alg_values_supported must include 'RS256'")
 	}
-	if contains(p.TokenEndpointAuthMethodsSupported, "none") {
```

Comment:
That seems redundant because of https://github.com/coreos/go-oidc/blob/master/oidc/provider.go#L353

If `RS256` isn't present in signing alg an error is returned. e.g. having `id_token_signing_alg_values_supported : [ "none" ]` will still return an error.